web/svelte.config.js

```diff
@@ -9,7 +9,7 @@ process.env.PUBLIC_IMMICH_PAY_HOST = process.env.PUBLIC_IMMICH_PAY_HOST || 'http
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
-  preprocess: vitePreprocess(),
+  preprocess: vitePreprocess({ script: true }),
```

Comment:
This is needed to have enums in svelte components. (We probably shouldn't and instead move them to `.ts` files, but I only made the current state work)

Reply:
FYI - this is the perfect use case and reason for a TODO: comment